### PR TITLE
FastQC and Foxit

### PR DESCRIPTION
**Pull request by gyengen**

### CODE REVIEW

File: sharc/software/apps/FastQC.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+   :Dependencies: Java for GUI. Module loaded for Java 1.8.0_102.
+   :URL: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/
+   :Documentation: https://www.bioinformatics.babraham.ac.uk/projects/fastqc/Help/
+
+FastQC aims to provide a simple way to do some quality control checks on raw sequence data coming from high throughput sequencing pipelines. It provides a modular set of analyses which you can use to give a quick impression of whether your data has any problems of which you should be aware before doing any further analysis.
+
+Usage
+-----
+
+FastQC version can be activated using the module files::
```
Comment (dmrogers75) [inline sharc/software/apps/FastQC.rst]: There is only one module file. Change "files" to "file".

File: sharc/software/apps/FastQC.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+	module load apps/FastQC/0.11.7/binary
+
+Then run using ``fastqc``.
```
Comment (dmrogers75) [inline sharc/software/apps/FastQC.rst]: Then run using the command ``fastqc``.

File: sharc/software/apps/FastQC.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Installation notes
+------------------
+
+FastQC was installed using the
+:download:`install_fastqc.sh </sharc/software/install_scripts/apps/FastQC/0.11.7/install_fastqc.sh>` script, the module
+file is
+:download:`0.11.7 </sharc/software/modulefiles/apps/FastQC/0.11.7>`.
```
Comment (dmrogers75) [inline sharc/software/apps/FastQC.rst]: Consider having full path to module file as link text and "binary" is missing. I.e. /usr/local/modulefiles/apps/FastQC/0.11.7/binary

File: sharc/software/apps/foxit.rst [hunk 4]
```diff
@@ -5,9 +5,9 @@ Foxit
 .. sidebar:: Foxit
    
    :Version: 2.4.0
-   :Dependancies:
-   :URL: http://cdn01.foxitsoftware.com/pub/foxit/reader/desktop/linux/2.x/2.4/en_us/FoxitReader2.4.0.14978_Server_x64_enu_Setup.run.tar.gz
-   :Documentation: https://help.foxitsoftware.com/manuals/pdf-reader/foxit-reader-for-mac-linux/en-us/2.4/
+   :Dependancies: -
```
Comment (dmrogers75) [inline sharc/software/apps/foxit.rst]: Dependencies.

File: sharc/software/modulefiles/apps/FastQC/0.11.7 [hunk 5]
```diff
@@ -0,0 +1,15 @@
+#%Module1.0#####################################################################
+
+## Module file logging
+source /usr/local/etc/module_logging.tcl
+
+proc ModulesHelp { } {
+    global version
+    puts stderr "   Makes FastQC $version available for use"
```
Comment (dmrogers75) [inline sharc/software/modulefiles/apps/FastQC/0.11.7]: $version is not defined.

File: sharc/software/modulefiles/apps/FastQC/0.11.7 [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+}
+module-whatis   "Makes FastQC available for use"
```
Comment (dmrogers75) [inline sharc/software/modulefiles/apps/FastQC/0.11.7]: Add $version as in above print statement.

File: sharc/software/apps/FastQC.rst (hunk 1 above)
Comment (gyengen) [reply sharc/software/apps/FastQC.rst]: Fixed.

File: sharc/software/apps/FastQC.rst (hunk 2 above)
Comment (gyengen) [reply sharc/software/apps/FastQC.rst]: Thanks, fixed.

**Pull request merged**